oauth2-server/src/main/java/com/clouway/oauth2/Identity.java

```diff
@@ -53,6 +56,32 @@ public String picture() {
     return claims;
   }
 
+  public JsonObject asJson(){
```

Comment:
I prefer to be moved out of this class instead of putting it in there, as it adds explicit JsonObject dependency in client code.  (accounts will be able to use it, which doesn't sounds reasonable)

UserInfoResponse response = UserInfoResponse.of(identity); //create user info from identity
response.asJson();


where the UserInfo will be packed as private class which will be visible only internally by the library.